packages/mobile/src/onboarding/contacts/ImportContactsScreen.tsx

```diff
@@ -178,8 +178,9 @@ class ImportContactScreen extends React.Component<Props, State> {
           )}
           {status > ImportContactsStatus.Stopped && this.renderImportStatus()}
         </ScrollView>
-        <View style={styles.bottomButtonContainer}>
+        <View style={[styles.bottomButtonContainer, isSkipHidden ? styles.invisible : undefined]}>
```

Comment:
I see, that `StyleProp` accepts `type Falsy = undefined | null | false;`, so could we use this one instead?
```suggestion
        <View style={[styles.bottomButtonContainer, isSkipHidden && styles.invisible]}>
```

Reply:
Good idea!